### DISCUSSION TITLE
Replit leaving the Student Developer Pack

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#81179] [GitHub Education]
### Hi everyone,

![new_logo](https://github.com/community/community/assets/107881423/d5634096-3f6b-4b84-a4c2-2a89ffc3ba6e)

### **TL; DR: Replit is leaving the Student Developer Pack.** 

Replit's student offer was 6 months of free private repls for verified students. Any existing students who have redeemed the offer to kick off that 6 month term will not experience a disruption in service. You can still redeem your offer by going to [Replit's student landing page](https://replit.com/site/github-students) until 3pm Friday EST. Any students attempting to redeem the offer after Friday 12.22.23 will be encouraged to proceed with a free Replit account, which still gives plenty of features and support, but does not give free private repls.

**FAQ's**
What if I've already redeemed my offer and started using Replit?
- If you have already redeemed your Replit offer, you will retain your benefits and not experience a disruption in service (e.g. if you are on month 3, you still have 3 more months of private repl access). 

What if I'm a new student and have not yet redeemed my Replit student offer?
- You have until Friday, December 22nd, 12pm PT to redeem the offer. Beyond that, Replit can still be used for free, but without access to private repls via this offer.

Replit service problems or questions?
- You can contact [support@replit.com](mailto:support@replit.com), or check out their support page [here](https://docs.replit.com/legal-and-security-info/support-policy). Please note that Student Developer Pack offers from third party companies manage their own support for their products and tools.

Looking for other IDE's in the Student Developer Pack?
- You can try Visual Studio, Azure and Jetbrains as alternatives within the pack, plus you can still sign up for the free version of Replit.

**Comment by manik-007** [comment]
Hello @morganersery1 

First of all thankyou for letting us know about this. I have made an account on Replit using the link mentioned in your article but still when i want to make private repl its asking for payment or card details?
But according to the information mentioned above, it should not be the case until Friday.

Kindly resolve my issue and help me to claim this previledge.
I would be very thankful to you.

Looking forward to a positive and quick response.

Regards ~ Manik

**Reply by morganersery1** [reply]
@manik-007 Please contact Replit support to have a ticket created in their system. I can confirm that there is nothing wrong with your account on our end, in case they ask.

**Comment by manik-007** [comment]
@morganersery1  Thanks for your quick response. I have raised a ticket there as you've told me now waiting for thier response.

**Comment by Bardin08** [comment]
@morganersery1, it makes sense to correct the announcement because Replit already redirects directly to the pricing page from https://replit.com/site/github-students.

That's really sad, and that Replit decided to leave.

**Reply by morganersery1** [reply]
@Bardin08 Thank you! We'll get in touch with them to see if they'll be able to honor the final day of student offer.

**Comment by Tamaramurali** [comment]
First of all thankyou for letting us know about this. I have made an account on Replit using the link mentioned in your article but still when i want to make private repl its asking for payment or card details?
